docker/Dockerfile

```
@@ -60,19 +60,16 @@ RUN docker-php-ext-enable mysqli curl gettext sockets pdo_mysql zip pcntl intl o
 
-#COPY --from=golang /usr/local/go/ /usr/local/go/
-
 ENV APP_VERSION="1.0.0"
-ENV VERSION="v2025.1.1"
+ENV VERSION="v2025.1.2"
 
-COPY docker/velox_rr_2024.toml .
+COPY docker/velox_rr.toml .
 
 ENV CGO_ENABLED=0
 RUN --mount=type=secret,id=RT_TOKEN,env=RT_TOKEN \
```

Hadolint (2.12.0)

[error] unexpected "env=RT_TOKEN \"
expecting "gid=", "id=", "mode=", "required", "required=False", "required=True", "required=false", "required=true", "uid=", source=, or target=

(DL1000)